outlines/generate/__init__.py

```diff
@@ -13,9 +14,6 @@
 from .regex import regex
 from .text import text
 
-if TYPE_CHECKING:
```

Comment:
What was the point of that, and why did you remove it?

Reply:
I think that was a mistake as we need to import `RegexLogitsProcessor` to use it in the `LocalGenerator`